workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -378,28 +374,19 @@ def _apply_physics(self) -> Diagnostics:
             "total_precip_after_physics": self._state[TOTAL_PRECIP],
         }
 
-    def _print_timing(self, name, min_val, max_val, mean_val):
-        self._print(f"{name:<30}{min_val:15.4f}{max_val:15.4f}{mean_val:15.4f}")
-
-    def _print_global_timings(self, root=0):
+    def log_global_timings(self, root=0):
         is_root = self.rank == root
         recvbuf = np.array(0.0)
         reduced = {}
-        self._print("-----------------------------------------------------------------")
```

Comment:
why not output both human-readable and machine-readable timings as is currently done?

Reply:
Sure, that's fine